llvm/lib/Target/AMDGPU/SIInstructions.td

```diff
@@ -546,6 +546,10 @@ def SI_MASKED_UNREACHABLE : SPseudoInstSI <(outs), (ins),
   let maybeAtomic = 0;
 }
 
+def SI_READANYLANE : SPseudoInstSI <(outs SReg_32:$dst), (ins VGPR_32:$src)> {
+  let SALU = 1;
```

Comment:
Should be a VALU instruction like readfirstlane is. Maybe define it with `VPseudoInstSI`?